tests/pytests/test_monitor_top.py

```diff
@@ -0,0 +1,93 @@
+# =============================================================================
+# 2015+ Copyright (c) Budnik Andrey <budnik27@gmail.com>
+# All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# =============================================================================
+
+import sys
+sys.path.insert(0, "")  # for running from cmake
+from conftest import make_session
+import pytest
+import elliptics
+import zlib
+import json
+import hashlib
+try:
+    import urllib.request as urllib_req
+except ImportError:
+    import urllib2 as urllib_req
+
+
+def get_top_by_http(remote, port):
+    url = 'http://' + remote + ':' + port + '/top'
+    data = urllib_req.urlopen(url).read()
+    json_data = zlib.decompress(data)
+    return json.loads(json_data)
+
+def has_key(key, keys):
+    for k in keys:
+        if k['id'] == key:
+            return True
+    return False
+
+def check_keys_fields(keys):
+    '''all items in top key list must contain group_id, key_id, traffic size generated by key access, key access frequency'''
+    for k in keys:
+        assert k['group']
+        assert k['id']
+        assert k['size']
+        assert k['frequency']
+
+'''
+Test fact of top keys service existance, service work and protocol correctness
```

Comment:
How this test does this? Please describe in more details what test is supposed to run, what results should be obtained and how they are going to be checked